nzsl-dictionary-scripts/env-production/main.tf

```
@@ -0,0 +1,71 @@
+
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "4.13.0"
+    }
+  }
+
+  backend "s3" {
+    bucket = "nzsl-infrastructure-terraform-state"
+    region = "ap-southeast-2"
+    key    = "nzsl-dictionary-scripts/env-production.tfstate"
+  }
+}
+
+provider "aws" {
+  region = "ap-southeast-2"
+}
```

suggestion: since this is a new env lets use `default_tags`:

```suggestion
  region = "ap-southeast-2"

  default_tags {
    tags = local.default_tags
  }
```